src/components/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { User } from '../../types/User';
+
+import './UserInfo.scss';
+
+type Props = {
+  user: User;
+};
```

Comment:
You got only one entity in your props, you can use it's type as your Props type
```suggestion
type Props = User;
```